cypress/spec/create_work_version.cy.js

```diff
@@ -1,4 +1,4 @@
-describe('Create work', () => {
+describe('Create work version', () => {
```

Comment:
Changed this since it's using the edit action, and therefore the work already exists. A new work version is what is getting created and tested.